pkg/avancement/service_manager_test.go

```diff
@@ -37,15 +37,15 @@ func TestPromoteWithSuccessCustomMsg(t *testing.T) {
 func promoteWithSuccess(t *testing.T, keepCache bool, repoType string, tlsVerify bool, msg string) {
 	dstBranch := "test-branch"
 	author := &git.Author{Name: "Testing User", Email: "testing@example.com", Token: "test-token"}
-	devRepo, stagingRepo := mock.New("/dev", "master"), mock.New("/staging", "master")
+	devRepo, stagingRepo := mock.New("/environments/dev", "master"), mock.New("/environments", "master")
```

Comment:
That's how I thought it worked, but no, the test thinks we have no files at all for the staging repository unless I follow it up with

`	stagingRepo.AddFiles("")`

You get this failure otherwise

```
^[[Abash-3.2$ go test ./...
?       github.com/rhd-gitops-example/services/cmd/services     [no test files]
--- FAIL: TestPromoteWithSuccessKeepCacheTrueWithGHE (0.00s)
    service_manager_test.go:68: could not determine unique environment name for destination repository, error: mock tried to get unique environment folder name but found no files for this repository, or several env folders: 
--- FAIL: TestPromoteWithSuccessKeepCacheFalseWithGitHub (0.00s)
    service_manager_test.go:68: could not determine unique environment name for destination repository, error: mock tried to get unique environment folder name but found no files for this repository, or several env folders: 
```